src/org/folio/rest_v2/Eureka.groovy

```diff
@@ -0,0 +1,98 @@
+package org.folio.rest_v2
+
+import org.folio.models.OkapiTenant
+import org.folio.models.RancherNamespace
+import org.folio.utilities.RequestException
+
+class Eureka extends Authorization {
+
+  public OkapiTenant superTenant
+
+  Eureka(Object context, String okapiDomain, OkapiTenant superTenant, boolean debug = false) {
+    super(context, okapiDomain, debug)
+    this.superTenant = superTenant
+  }
+
+  def createTenant(OkapiTenant tenant) {
+    String url = generateUrl("/tenants")
```

Comment:
This method should be type void as nothing return

Reply:
Adjusted